x/staking/types/msg.go

```diff
@@ -24,6 +24,7 @@ var (
 type MsgCreateValidator struct {
 	Description   Description    `json:"description"`
 	Commission    CommissionMsg  `json:"commission"`
+	MinSelfBond   sdk.Int        `json:"min_self_bond"`
```

Comment:
MinSelfDelegation is more explicit and correct, should find and replace for all variable names

Reply:
++

Reply:
Will do.  As a side note, we currently use the term "self bond" in distribution.  Maybe we should change that as well?  @cwgoes